test/specs/lwcLsp.e2e.ts

```diff
@@ -84,7 +91,15 @@ describe('LWC LSP', async () => {
     // Get open text editor
     const workbench = await browser.getWorkbench();
     const textEditor = await utilities.getTextEditor(workbench, 'lwc1.html');
-    await textEditor.typeTextAt(3, 7, ' lwc');
+
+    // Move cursor to right after the first 'div' tag and type ' lwc'
+    await browser.keys([CMD_KEY, 'f']);
+    await utilities.pause(1);
+    await browser.keys(['div']);
```

Comment:
Same here?

Reply:
This one cannot be combined because of the pause on Line 97 - that was an intentional pause to wait for the Find box to open.

Reply:
@daphne-sfdc I am confused. We have
await browser.keys(['div']);
await browser.keys(['Escape']);
await browser.keys(['ArrowRight']);
await utilities.pause(1);

Why cannot this become?
await browser.keys(['div','Escape','ArrowRight']);
await utilities.pause(1);

Reply:
Oh sorry!  Yes you're right, Github only previewed the first four lines, so I only saw `await browser.keys(['div']);` and didn't realize what you were referring to.